lib/renderer/init.js

```diff
@@ -141,7 +141,20 @@ if (nodeIntegration === 'true') {
 
   // Set the __filename to the path of html file if it is file: protocol.
   if (window.location.protocol === 'file:') {
-    var pathname = process.platform === 'win32' && window.location.pathname[0] === '/' ? window.location.pathname.substr(1) : window.location.pathname
+    let pathname
+    const loc = window.location
+    if (process.platform === 'win32') {
+      if (loc.pathname[0] === '/') {
+        pathname = loc.pathname.substr(1)
+      }
+
+      if (loc.hostname.length > 0 && globalPaths[0].startsWith('\\')) {
```

Comment:
It would be great to have a comment describing (or a link to) an issue this addresses.
Or a simple variable could help too:

```js
const isWindowsNetworkSharePath = location.hostname.length > 0 && globalPaths[0].startsWith('\\')
if (isWindowsNetworkSharePath) {
  ...
}
```